recipe/utils.py

```diff
@@ -153,8 +154,12 @@ def replace_whitespace_with_space(s):
 
 
 def clean_unicode(value):
+    """Convert a unicode value into ASCII bytes."""
     try:
-        cleaned_value = str(value)
+        if isinstance(value, string_types):
```

Comment:
I think we need `.encode("ascii", "ignore")` here

Reply:
We do, we do!

Reply:
should we also do this in fruition's version of this function?

actually, maybe we should delete fruition's version and have it use recipe.utils.clean_unicode :)